### PR TITLE
feat: inspection report

### PR DESCRIPTION
**Pull request by PhiNSE**

### CODE REVIEW

File: src/modules/inspection-request/dto/create-inspection-request.dto.ts
Comment (LongNHSE) [inline src/modules/inspection-request/dto/create-inspection-request.dto.ts:1]: Add decorators to check if staff or department exist

File: src/modules/inspection-request/dto/update-inspection-request.dto.ts
Comment (LongNHSE) [inline src/modules/inspection-request/dto/update-inspection-request.dto.ts:1]: Remove id from updatedDto, we should not update id

File: src/modules/inspection-request/inspection-request.service.ts [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    );
+    return inspectionRequest;
+  }
+
+  async findAll() {
+    return this.prismaService.inspectionRequest.findMany();
+  }
+
+  async findUnique(id: string) {
+    const inspectionRequest = this.prismaService.inspectionRequest.findUnique({
```
Comment (LongNHSE) [inline src/modules/inspection-request/inspection-request.service.ts:75]: Add await keyword to get result

**Review by LongNHSE — CHANGES_REQUESTED** [review]
Adjust some lines based on suggestion

File: src/modules/inspection-request/dto/update-inspection-request.dto.ts
Comment (PhiNSE) [reply src/modules/inspection-request/dto/update-inspection-request.dto.ts:1]: [Should not fix] Id is for bulk update  where { id } clause

File: src/modules/inspection-request/dto/create-inspection-request.dto.ts
Comment (PhiNSE) [reply src/modules/inspection-request/dto/create-inspection-request.dto.ts:1]: fixed in new commit

**Review by LongNHSE — COMMENTED** [review]
gud

**Review by LongNHSE — APPROVED** [review]

**Pull request merged**